src/main/java/io/cdap/plugin/format/MultiTableDBConfiguration.java

```diff
@@ -0,0 +1,57 @@
+package io.cdap.plugin.format;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.mapreduce.lib.db.DBConfiguration;
+
+import java.lang.reflect.Type;
+import java.util.List;
+
+/**
+ * Allows to specify and access connection configuration properties of {@link Configuration}.
+ */
+public class MultiTableDBConfiguration extends DBConfiguration {
+  private static final String PLUGIN_CONF_FIELD = "multi.db.plugin.conf";
+  private static final String DRIVER_FIELD = "multi.db.jdbc.connection";
+  private static final String INFO_FIELD = "multi.db.jdbc.dbinfo";
+
+  private static final Type PLUGIN_CONF_TYPE = new TypeToken<MultiTableConf>() {
+  }.getType();
```

Comment:
nit: can be on previous line: `{ }.getType()`

Reply:
done